Packs/SecneurXAnalysis/Integrations/SecneurXAnalysis/SecneurXAnalysis.py

```diff
@@ -0,0 +1,611 @@
+                                                          SNXResponse.SNX_STATUS_KEY, SNXResponse.SNX_SHA256_KEY])
+                return CommandResults(
+                    readable_output=readableOutput,
+                    outputs_prefix="",
+                    outputs={},
+                    raw_response=reportList
+                )
+            else:
+                msgJson = {'msg': "No samples to display"}
+                readableOutput = tableToMarkdown("SecneurX Analysis - List of Samples in Pending State:", msgJson)
+                return CommandResults(
+                    readable_output=readableOutput,
+                    outputs_prefix="",
+                    outputs=msgJson
+                )
+        else:
+            readableOutput = tableToMarkdown("", response)
+            return CommandResults(
+                readable_output=readableOutput,
+                outputs_prefix="",
+                outputs=response
+            )
+    else:
+        msg = error_response(err_msg)
+        raise DemistoException(msg)
+
+
+def get_status_cmd(client: Client, args: Dict[str, str]) -> CommandResults:
+    urlSuffix = "/get_status"
+    params = create_request_json(args)
+    response, err_msg = client.get_response(urlSuffix, params)
+    if response:
+        if SNXResponse.SNX_SUCCESS_KEY in response.keys() and response[SNXResponse.SNX_SUCCESS_KEY] == SNXResponse.SUCCESS:
+            reportList = response[SNXResponse.SNX_RESULT_KEY]
+            if reportList != SNXResponse.SNX_NULL_KEY and len(reportList) > 0:
+                for report in reportList:
+                    if SNXResponse.SNX_FILENAME_KEY in report.keys():
+                        report[SNXResponse.SAMPLE_KEY] = report[SNXResponse.SNX_FILENAME_KEY]
+                    elif SNXReportParser.JSON_URL in report.keys():
+                        report[SNXResponse.SAMPLE_KEY] = report[SNXReportParser.JSON_URL]
+                    else:
+                        continue
+                readableOutput = tableToMarkdown("SecneurX Analysis - Status of Submitted Samples:", t=reportList,
+                                                 headers=[SNXResponse.SNX_TASK_UUID_KEY, SNXResponse.SAMPLE_KEY,
+                                                          SNXResponse.SNX_STATUS_KEY, SNXResponse.SNX_SHA256_KEY])
+                return CommandResults(
+                    readable_output=readableOutput,
+                    outputs_prefix="",
+                    outputs={},
+                    raw_response=reportList
+                )
+            else:
+                msgJson = {"msg": "No samples to display"}
+                readableOutput = tableToMarkdown("SecneurX Analysis - Status of Submitted Samples: ", msgJson)
+                return CommandResults(
+                    readable_output=readableOutput,
+                    outputs_prefix="",
+                    outputs=msgJson
+                )
```

Comment:
This outputs will not be under a special key.
Consider removing outputs so it doesn't return to context, or add to output_prefix like this:

```suggestion
                return CommandResults(
                    readable_output=readableOutput,
                    outputs_prefix="SnXAnalysis.Status",
                    outputs=msgJson
                )
```